neqo-transport/src/connection/params.rs

```diff
@@ -468,14 +469,14 @@ impl ConnectionParameters {
         tps.local_mut()
             .set_integer(InitialMaxStreamsUni, self.max_streams_uni);
         tps.local_mut().set_integer(
-            TransportParameterId::IdleTimeout,
+            IdleTimeout,
             u64::try_from(self.idle_timeout.as_millis()).unwrap_or(0),
         );
         if let PreferredAddressConfig::Address(preferred) = &self.preferred_address {
             if role == Role::Server {
                 let (cid, srt) = cid_manager.preferred_address_cid()?;
                 tps.local_mut().set(
-                    TransportParameterId::PreferredAddress,
+                    PreferredAddressTp,
```

Comment:
Unrelated?

Reply:
While the patient was open.  I wanted to make the usage consistent.